--- SConstruct_old	2009-08-24 23:38:50.839134830 +0600
+++ SConstruct	2009-08-24 23:39:09.589132739 +0600
@@ -53,7 +53,6 @@
     basedir + 'Externals/LZO',
     basedir + 'Externals/SOIL',
     basedir + 'Externals/Lua',
-    basedir + 'Externals/WiiUseSrc/Src',
     basedir + 'Source/Core/VideoCommon/Src',
     basedir + 'Source/Core/InputCommon/Src',
     basedir + 'Source/Core/AudioCommon/Src',
@@ -66,7 +65,6 @@
     'Externals/LZO',
     'Externals/SOIL',
     'Externals/Lua',
-    'Externals/WiiUseSrc/Src', 
     'Source/Core/Common/Src',
     'Source/Core/Core/Src',
     'Source/Core/DiscIO/Src',
