diff --git a/zypp/parser/xml/Node.cc b/zypp/parser/xml/Node.cc
index e3ac4cd..c705774 100644
--- a/zypp/parser/xml/Node.cc
+++ b/zypp/parser/xml/Node.cc
@@ -9,11 +9,9 @@
 /** \file zypp/parser/xml/Reader.cc
  *
 */
-extern "C"
-{
+
 #include <libxml/xmlreader.h>
 #include <libxml/xmlerror.h>
-}
 
 #include <iostream>
 
diff --git a/zypp/parser/xml/Reader.cc b/zypp/parser/xml/Reader.cc
index 4e514f4..d06be45 100644
--- a/zypp/parser/xml/Reader.cc
+++ b/zypp/parser/xml/Reader.cc
@@ -9,11 +9,8 @@
 /** \file zypp/parser/xml/Reader.cc
  *
 */
-extern "C"
-{
 #include <libxml/xmlreader.h>
 #include <libxml/xmlerror.h>
-}
 
 #include <iostream>
 
