--- file_not_specified_in_diff	
+++ file_not_specified_in_diff	
@@ -17,1 +17,1 @@ 
-IUSE="nsplugin linguas_de linguas_es linguas_fr linguas_ja linguas_zh_CN"
---
+IUSE="nsplugin minimal linguas_de linguas_es linguas_fr linguas_ja linguas_zh_CN"
@@ -24,0 +24,2 @@ 
+# Minimal avoids installing selfservice utility and its dependencies
+# as well as optional multimedia dependencies (cf. deb package)
@@ -35,4 +37,0 @@ 
-	media-libs/gst-plugins-base:0.10
-	media-libs/gstreamer:0.10
-	media-libs/libcanberra[gtk]
-	media-libs/libogg
@@ -40,4 +38,1 @@ 
-	media-libs/libvorbis
-	media-libs/speex
-	net-libs/webkit-gtk:2
-	virtual/krb5
---
+	net-dns/libidn
@@ -48,1 +43,0 @@ 
-	x11-libs/libXaw
@@ -52,0 +46,1 @@ 
+	x11-libs/libXpm
@@ -53,2 +48,13 @@ 
-	x11-libs/libXt
-	x11-libs/pango"
---
+	x11-libs/pango
+	!minimal? (
+		media-libs/gst-plugins-base:0.10
+		media-libs/gstreamer:0.10
+		media-libs/libcanberra[gtk]
+		media-libs/libogg
+		media-libs/libvorbis
+		media-libs/speex
+		net-libs/webkit-gtk:2
+		virtual/krb5
+		x11-libs/libXaw
+		x11-libs/libXt
+	)"
@@ -85,1 +91,4 @@ 
-	doexe *.DLL libctxssl.so libproxy.so wfica AuthManagerDaemon PrimaryAuthManager selfservice ServiceRecord
---
+	doexe *.DLL libctxssl.so libproxy.so wfica AuthManagerDaemon PrimaryAuthManager 
+	if ! use minimal ; then
+		doexe selfservice ServiceRecord
+	fi

