diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index 5b55cd9..5ac72f4 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -178,7 +178,7 @@ set_target_properties(zmusic zmusiclite PROPERTIES
 	SOVERSION ${ZMUSIC_VERSION_MAJOR}
 )
 
-install(TARGETS zmusic zmusiclite LIBRARY DESTINATION lib)
+install(TARGETS zmusic zmusiclite LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 install(DIRECTORY ../include/ DESTINATION include FILES_MATCHING PATTERN "*.h*")
 
 source_group("MIDI Devices" REGULAR_EXPRESSION "^${CMAKE_CURRENT_SOURCE_DIR}/mididevices/.+")
